python/py-coveralls/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-coveralls
+python.rootname     python-coveralls
+version             2.9.3
+revision            0
+categories-append   devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python API for http://coveralls.io.
+
+long_description    ${description}.
+
+homepage            https://github.com/z4r/python-coveralls
+
+master_sites        pypi:p/${python.rootname}/
```

Comment:
The python port group sets this master_sites value by default, so you need not specify it explicitly.

Reply:
Done thanks.